_data-prepper/trace-analytics.md

```diff
@@ -0,0 +1,377 @@
+        # username and password above.
+        #aws_sigv4: true
+        #aws_region: us-east-1
+```
+
+You need to modify the preceding configuration for your OpenSearch cluster. Note that it has two `opensearch` sinks which need to be modified.
+{: .note}
+
+You must make the following changes:
+* `hosts` - Set to your hosts
+* `username` - Provide the OpenSearch username.
+* `password` - Provide your OpenSearch password.
+* `aws_sigv4` - If you are using Amazon OpenSearch Service with AWS signing, set this value to `true`. It will sign requests with the default AWS credentials provider.
+* `aws_region` - If you are using Amazon OpenSearch Service with AWS signing, set this value to your region.
+
+For other configurations available for OpenSearch sinks, see [Data Prepper OpenSearch Sink]({{site.url}}{{site.baseurl}}/data-prepper/configuration/sinks/sinks/).
+
+## OpenTelemetry Collector
+
+You need to run OpenTelemetry collector in your service environment. Follow the [getting started guide](https://opentelemetry.io/docs/collector/getting-started/#getting-started) to install an OpenTelemetry collector.  Ensure that you configure the collector with an exporter configured to your Data Prepper. The following example `otel-collector-config.yaml` receives data from various instrumentations and exports it to Data Prepper.
+
+### Example otel-collector-config.yaml file
+
+The following is an example `otel-collector-config.yaml` file:
+
+```
+receivers:
+  jaeger:
+    protocols:
+      grpc:
+  otlp:
+    protocols:
+      grpc:
+  zipkin:
+
+processors:
+  batch/traces:
+    timeout: 1s
+    send_batch_size: 50
+
+exporters:
+  otlp/data-prepper:
+    endpoint: localhost:21890
+    tls:
+      insecure: true
+
+service:
+  pipelines:
+    traces:
+      receivers: [jaeger, otlp, zipkin]
+      processors: [batch/traces]
+      exporters: [otlp/data-prepper]
+```
+
+After you run OpenTelemetry in your service environment, you must configure your application to use the OpenTelemetry collector. The OpenTelemetry collector typically runs alongside your application.
+
+## Next steps and more information
+
+The [OpenSearch Dashboards Observability plugin]({{site.url}}{{site.baseurl}}/observability-plugin/trace/ta-dashboards/) documentation provides additional details on configuring OpenSearch for viewing trace analytics in OpenSearch Dashboards.
+
+For more information about how to tune and scale Data Prepper for trace analytics, see [trace tuning](#trace-tuning).
+
+## Migrating to Data Prepper 2.0
+
+Starting in Data Prepper version 1.4, trace processing uses Data Prepper's event model. This allows pipeline authors to configure other processors to modify spans or traces. To provide a migration path, Data Prepper version 1.4 introduced the following changes:
+
+* The `otel_trace_source` has an optional parameter `record_type` which can be set to `event`. When configured, it will output event objects.
```

Comment:
```suggestion
* `otel_trace_source` has an optional `record_type` parameter that can be set to `event`. When configured, it will output event objects.
```